core/templates/pages/contributor-dashboard-page/voiceover-opportunities/voiceover-opportunities.component.ts

```diff
@@ -77,6 +78,7 @@ angular.module('oppia').component('voiceoverOpportunities', {
               ContributionOpportunitiesService.getVoiceoverOpportunities(
                 TranslationLanguageService.getActiveLanguageCode(),
                 updateWithNewOpportunities);
```

Comment:
Just to be safe, put a .then() and add $rrotScope.$applyAsync inside.

Reply:
@srijanreddy98 I made this change
```
ContributionOpportunitiesService.getVoiceoverOpportunities(
TranslationLanguageService.getActiveLanguageCode(),
updateWithNewOpportunities).then($rootScope.$applyAsync());
```

 but it was giving error `cannot read property then of undefined`